Dockerfile.release

```diff
@@ -1,12 +1,24 @@
 FROM alpine:latest as alpine
 RUN apk add -U --no-cache ca-certificates
 
```

Comment:
We don't need this @nitisht - we can use only one layer here i.e RedHat UBI image